### PR TITLE
Update passwordless setup of Postgres for Windows

### PR DESCRIPTION
**Pull request by harshitbansal05**
This pull request aims to fix: #1230. It adds instructions for `pgpass.conf` on Microsoft Windows.

### CODE REVIEW

**Comment by henrykironde** [comment]
Looks good to me. @ethanwhite you went through some windows setup recently, what do you think about this?

File: docs/developer.rst [hunk 1]
```diff
@@ -99,14 +99,19 @@ links `Read The Docs`_, `codecov`_, `AppVeyor`_ and  `Travis`_
...
 --------------------------
 
 To avoid supplying the passwords when using the tool, use the config files
-`.pgpass` for Postgres and `.my.cnf` for MySQL. The files are kept in the
-HOME directory(~/.pgpass, ~/.my.cnf). Make sure you set the file permissions to 600.
+`.pgpass`(`pgpass.conf` for Microsoft Windows) for Postgres and `.my.cnf`
+for MySQL. The files are kept in the HOME directory(~/.pgpass, ~/.my.cnf).
+Make sure you set the file permissions to 600. For Postgres, on Microsoft
+Windows, entering `%APPDATA%` will take you to `C:\Users\username\AppData\Roaming`.
+Once you get there you need to create the `postgresql directory` and then create
```
Comment (ethanwhite) [inline docs/developer.rst]: I'd phrase this as "In this directory create a new subdirectory named `postgresql`. Then create the pgpass.conf file inside it."

**Comment by harshitbansal05** [comment]
@henrykironde @ethanwhite could you please review it now?

**Comment by henrykironde** [comment]
Looks good. Could you rebase with master.

**Comment by harshitbansal05** [comment]
@henrykironde I had already rebased it.

**Comment by henrykironde** [comment]
Oh Great, I was looking at the forced push diff. 
In it goes.

**Pull request merged**